src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java

```diff
@@ -46,10 +46,7 @@
 import com.google.inject.Provider;
 
 import org.rstudio.core.client.*;
-import org.rstudio.core.client.command.AppCommand;
-import org.rstudio.core.client.command.Handler;
-import org.rstudio.core.client.command.KeyboardShortcut;
-import org.rstudio.core.client.command.ShortcutManager;
+import org.rstudio.core.client.command.*;
```

Comment:
We generally don't use wildcard imports (there are plenty scattered around in the code so not a huge deal). IntelliJ does them by default but there's a setting to override.

Reply:
Oh yes, I think IntelliJ changed this on me. Let me change it back, and I'll update that setting too.